daemon/top_windows.go

```diff
@@ -0,0 +1,11 @@
+package daemon
```

Comment:
_windows is recognized by golang as a file postfix, where-as _unix is not. To allow for FreeBSD, more and more _Linux files (which is a recognized postfix) are being changed to _unix. Hence _unix needs the build tag.